tools/setup.py

```diff
@@ -15,14 +15,14 @@
     " please refer to the further documentation.",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/fsoco/fsoco",
+    url="https://www.fsoco-dataset.com",
```

Comment:
```suggestion
    url="https://www.fsoco-dataset.com/tools/",
```
For more information on the tool suite.